isup/isup-impl/src/main/java/org/mobicents/protocols/ss7/isup/util/BcdHelper.java

```diff
@@ -0,0 +1,119 @@
+/**
+ *  Copyright 2016 Grzegorz Figiel
+ *
```

Comment:
Then you can change the license header. but do keep the comments on what was changed over the original Apache license file.